CMakeLists.txt

```diff
@@ -8,10 +8,11 @@ include_directories(${PROJECT_SOURCE_DIR}/src/include)
 add_subdirectory(googletest/)
 
 set(SOURCE_FILES
-        src/lexer/Lexer.cpp)
+        src/lexer/Lexer.cpp
+        src/parser/Parser.cpp)
 
 set(TEST_FILES
-        test/TestLexer.cpp)
+        test/TestLexer.cpp test/TestParser.cpp src/include/ASTree.h)
```

Comment:
You don't need to include `.h` files in the CMakeLists.

Reply:
ah, i was just blindly hitting "ok" on the CLion prompts